src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -299,6 +348,24 @@ public ColumnMask getMask()
         return mask;
     }
 
+    @Nonnull
+    public ColumnConstraints getColumnConstraints()
+    {
+        return columnConstraints;
+    }
+
+    @Nonnull
+    public void setColumnConstraints(ColumnConstraints constraints)
+    {
+        this.columnConstraints = constraints;
+    }
+
+    @Nullable
+    public void removeColumnConstraints()
+    {
+        columnConstraints = new ColumnConstraints.Noop();
```

Comment:
`new ColumnConstraints.Noop()` can be a singleton. You do not need to create new instances of this every time you are going to deal with this.